uaclient/apt.py

```diff
@@ -104,6 +147,18 @@ def run_apt_command(
     except util.ProcessExecutionError as e:
         if "Could not get lock /var/lib/dpkg/lock" in str(e.stderr):
             error_msg += " Another process is running APT."
+        else:
+            """
+            Treat errors where one of the APT repositories
+            is invalid or unreachable. In that situation, we alert
+            which repository is causing the error
+            """
```

Comment:
This docstring should be inside _parse_apt_pdate_for_invalid_apt_config function. It doesn't really belong up here.

Reply:
Maybe we could keep it where it is, since this docstring is describing why we are doing the checks before calling the `_parse_apt_update_for_invalid_apt_config`. But if you disagree, I can move it inside the parser function

Reply:
+1 we can leave this docstring.
I still think we should drop the 3 lines below this docstring because that logic is contained within _parse_apt_update_for_invalid_apt_config which will return "" if no matching rep_error or fetch_error.

So I think those lines below can be just
error_msg += _parse_apt_update_for_invalid_apt_config(e.stderr)